[docs] CDC: Update content for CDC streams and transactional ordering (#25112)
yugabyte/yugabyte-db: docs/content/preview/explore/change-data-capture/using-yugabytedb-grpc-replication/cdc-monitor.md

   curl -X GET localhost:8083/connectors/<connector-name>/status
   ```

{{< note title= "Note" >}}

> It is possible, that upon retrieving the status of the connector, it can show a `RUNNING` state while no data is being ingested to Kafka. As a connector consists of one or more tasks, it is likely that the tasks have failed, independently of the connector. To verify this, you'll need to check for the status of the tasks.<br/><br/>The status of the task is present in the response of the following command:

Reviewer: ```suggestion
It is possible that upon retrieving the status of the connector, it can show a RUNNING state while no data is being ingested to Kafka. As a connector consists of one or more tasks, the tasks have likely failed, independently from the connector. To verify this, you need to check for the status of the tasks.

To show the status, use the following command:
```

```sh
curl -X GET localhost:8083/connectors/<connector-name>/status
```

> You can also get the status of a specific task using its task-id:

Reviewer: ```suggestion
You can also get the status of a specific task by passing in the task ID as follows:
```

```sh
curl -X GET localhost:8083/connectors/<connector-name>/tasks/<task-id>/status
```

{{ < /note > }}

> 

Reviewer 2: Same goes for this, it is also not rendering corrently.
